app/models/org.rb

```diff
@@ -170,6 +198,10 @@ def self.human_attribute_name(attr, options = {})
     HUMANIZED_ATTRIBUTES[attr.to_sym] || super
   end
 
+  def links
+    super() || { "org": [] }
+  end
```

Comment:
Is this to set the default value for forms in-case the value is nil?

Reply:
yes. I encountered some errors with newly created records. Since everyone's tables likely have records with a nil there I thought it safer to do on the getter instead of the setter. We could update the setter though and add an upgrade rake task